modules/tinymce/src/core/main/ts/api/html/Serializer.ts

```diff
@@ -89,7 +89,7 @@ const HtmlSerializer = (settings?: HtmlSerializerSettings, schema = Schema()): H
 
       if (!handler) {
         const name = node.name;
-        const isEmpty = node.shortEnded;
+        const isEmpty = name in schema.getShortEndedElements();
```

Comment:
I had completely forgotten you could use `in` like this. nice.

Reply:
It's normally a bad idea though and we should be using `Obj.has` generally though, but as this is parser related it's fine here.